File: Templates/Frameworks-Samples/Csla/CSharp/PetShop/PetShop.UI/Controls/CartList.ascx.cs
using System;

using PetShop.Business;

namespace PetShop.UI.Controls
{
    public partial class CartList : System.Web.UI.UserControl
    {
        /// <summary>
        /// Bind control
        /// </summary>
        public void Bind(Business.CartList cart)
        {
            if (cart != null)
            {
                repOrdered.DataSource = cart;
                repOrdered.DataBind();
            }

        }
    }
}

File: Framework-Samples/Csla/VisualBasic/PetShop.Silverlight/Source/PetShop.UI/Controls/CartList.ascx.vb
Public Partial Class CartList
    Inherits System.Web.UI.UserControl

    ''' <summary>
    ''' Bind control
    ''' </summary>
    Public Sub Bind(ByVal cart As Business.CartList)
        If Not cart Is Nothing Then
            repOrdered.DataSource = cart
            repOrdered.DataBind()
        End If

    End Sub

End Class